webservices/resources/filings.py

```diff
@@ -76,3 +76,40 @@ class FilingsList(BaseFilings):
     @property
     def args(self):
         return utils.extend(super().args, args.entities)
+
+
+@doc(
+    tags=['efiling'],
+    description=docs.EFILE_FILES,
+)
+class EFilingsView(views.ApiResource):
+
+    model = models.EFilings
+    schema = schemas.EFilingsSchema
+    page_schema = schemas.EFilingsPageSchema
+
+    filter_multi_fields = [
+        ('file_number', models.Filings.file_number),
+        ('committee_id', models.Filings.committee_id),
+    ]
+
+    @property
+    def args(self):
+        return utils.extend(
+            args.paging,
+            args.efilings,
+            args.make_sort_args(
+                default='-receipt_date',
+                # TODO: add indexes and turn on validation
+                # validator=args.IndexValidator(models.EFilings),
```

Comment:
I am not opposed to adding that. 

We should double check it looks at the actual sql, and not just our sqlalchemy annotations.